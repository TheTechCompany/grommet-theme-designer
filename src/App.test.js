import React from 'react';
import { cleanup, render, fireEvent } from '@testing-library/react';
import App from './App';

describe('App', () => {
  afterEach(cleanup);

  test('empty', () => {
    const { getByTitle, container } = render(<App />);
    expect(container.firstChild).toMatchSnapshot();

    fireEvent.click(getByTitle('choose another theme'));
    expect(container.firstChild).toMatchSnapshot();
  });
});
