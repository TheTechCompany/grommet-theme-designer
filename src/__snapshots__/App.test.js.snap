// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App empty 1`] = `
<div
  class="StyledGrommet-sc-19lkkz7-0 jKcNfZ"
>
  <div
    class="StyledGrid-sc-1wofa1l-0 bIzLvh"
  >
    <div
      class="StyledBox-sc-13pk1d4-0 vbgwq"
    >
      <div
        class="StyledBox-sc-13pk1d4-0 eYDYam"
      >
        <span
          class="StyledText-sc-1sadyjn-0 gmtNWw"
        >
          my theme
        </span>
        <button
          class="StyledButton-sc-323bzc-0 kAlxhW"
          title="edit theme"
          type="button"
        >
          <svg
            aria-label="Edit"
            class="StyledIcon-ofa7kd-0 kPmYvD"
            viewBox="0 0 24 24"
          >
            <path
              d="M14,4 L20,10 L14,4 Z M22.2942268,5.29422684 C22.6840146,5.68401459 22.6812861,6.3187139 22.2864907,6.71350932 L9,20 L2,22 L4,15 L17.2864907,1.71350932 C17.680551,1.319449 18.3127724,1.31277239 18.7057732,1.70577316 L22.2942268,5.29422684 Z M3,19 L5,21 M7,17 L15,9"
              fill="none"
              stroke="#000"
              stroke-width="2"
            />
          </svg>
        </button>
      </div>
      <div
        class="StyledBox-sc-13pk1d4-0 eXTSqT"
      >
        <div
          class="StyledBox-sc-13pk1d4-0 jJdKAu"
        >
          <label
            class="StyledRadioButton__StyledRadioButtonContainer-g1f6ld-0 eRuGHl"
            for="light"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 iOJnkv StyledRadioButton-g1f6ld-4 eLBYmH"
            >
              <input
                checked=""
                class="StyledRadioButton__StyledRadioButtonInput-g1f6ld-1 hBCEfd"
                id="light"
                name="themeMode"
                type="radio"
                value="light"
              />
              <div
                class="StyledBox-sc-13pk1d4-0 jxeStE StyledRadioButton__StyledRadioButtonBox-g1f6ld-3 fNAQJr"
              >
                <svg
                  class="StyledRadioButton__StyledRadioButtonIcon-g1f6ld-2 iHxKuy"
                  preserveAspectRatio="xMidYMid meet"
                  viewBox="0 0 24 24"
                >
                  <circle
                    cx="12"
                    cy="12"
                    r="6"
                  />
                </svg>
              </div>
            </div>
            <span>
              light
            </span>
          </label>
          <div
            class="StyledBox__StyledBoxGap-sc-13pk1d4-1 fskCS"
          />
          <label
            class="StyledRadioButton__StyledRadioButtonContainer-g1f6ld-0 eRuGHl"
            for="dark"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 iOJnkv StyledRadioButton-g1f6ld-4 eLBYmH"
            >
              <input
                class="StyledRadioButton__StyledRadioButtonInput-g1f6ld-1 hBCEfd"
                id="dark"
                name="themeMode"
                type="radio"
                value="dark"
              />
              <div
                class="StyledBox-sc-13pk1d4-0 kmXmMI StyledRadioButton__StyledRadioButtonBox-g1f6ld-3 fNAQJr"
              />
            </div>
            <span>
              dark
            </span>
          </label>
        </div>
      </div>
      <div
        class="StyledBox-sc-13pk1d4-0 llKfmK"
      >
        <div
          class="StyledBox-sc-13pk1d4-0 eXTSqT"
        >
          <h3
            class="StyledHeading-sc-1rdh4aw-0 jOHoby"
          >
            Guides
          </h3>
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 hFGUuI"
          tabindex="0"
        >
          Accessibility
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 hFGUuI"
          tabindex="0"
        >
          Typography
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 eXTSqT"
        >
          <h3
            class="StyledHeading-sc-1rdh4aw-0 jOHoby"
          >
            Examples
          </h3>
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 hFGUuI"
          tabindex="0"
        >
          Dashboard
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 hFGUuI"
          tabindex="0"
        >
          List
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 hFGUuI"
          tabindex="0"
        >
          Form
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 hFGUuI"
          tabindex="0"
        >
          Document
        </div>
      </div>
    </div>
    <div
      class="StyledBox-sc-13pk1d4-0 jJdKAu"
    />
    <div
      class="StyledBox-sc-13pk1d4-0 jiNNxm"
    >
      <div
        class="StyledBox-sc-13pk1d4-0 hTZIev"
      >
        <div
          class="StyledBox-sc-13pk1d4-0 koJzMg"
        >
          <button
            class="StyledButton-sc-323bzc-0 kAlxhW"
            title="choose another theme"
            type="button"
          >
            <svg
              aria-label="Apps"
              class="StyledIcon-ofa7kd-0 kPmYvD"
              viewBox="0 0 24 24"
            >
              <path
                d="M19,5 L21,5 L21,3 L19,3 L19,5 Z M11,5 L13,5 L13,3 L11,3 L11,5 Z M3,5 L5,5 L5,3 L3,3 L3,5 Z M19,13 L21,13 L21,11 L19,11 L19,13 Z M11,13 L13,13 L13,11 L11,11 L11,13 Z M3,13 L5,13 L5,11 L3,11 L3,13 Z M19,21 L21,21 L21,19 L19,19 L19,21 Z M11,21 L13,21 L13,19 L11,19 L11,21 Z M3,21 L5,21 L5,19 L3,19 L3,21 Z"
                fill="none"
                stroke="#000"
                stroke-width="2"
              />
            </svg>
          </button>
          <div
            class="StyledBox__StyledBoxGap-sc-13pk1d4-1 hUZSMI"
          />
          <div
            class="StyledBox-sc-13pk1d4-0 eitMOG"
          >
            <button
              class="StyledButton-sc-323bzc-0 jeaTYv"
              disabled=""
              title="undo last change"
              type="button"
            >
              <svg
                aria-label="Undo"
                class="StyledIcon-ofa7kd-0 kPmYvD"
                viewBox="0 0 24 24"
              >
                <path
                  d="M7.18,4,8.6,5.44,6.06,8h9.71a6,6,0,0,1,0,12h-2V18h2a4,4,0,0,0,0-8H6.06L8.6,12.51,7.18,13.92,2.23,9Z"
                />
              </svg>
            </button>
            <button
              class="StyledButton-sc-323bzc-0 jeaTYv"
              disabled=""
              title="redo last change"
              type="button"
            >
              <svg
                aria-label="Redo"
                class="StyledIcon-ofa7kd-0 kPmYvD"
                viewBox="0 0 24 24"
              >
                <path
                  d="M16.82,4,15.4,5.44,17.94,8H8.23a6,6,0,0,0,0,12h2V18h-2a4,4,0,0,1,0-8h9.71L15.4,12.51l1.41,1.41L21.77,9Z"
                />
              </svg>
            </button>
          </div>
          <div
            class="StyledBox__StyledBoxGap-sc-13pk1d4-1 hUZSMI"
          />
          <button
            class="StyledButton-sc-323bzc-0 kAlxhW"
            title="see JSON"
            type="button"
          >
            <svg
              aria-label="Code"
              class="StyledIcon-ofa7kd-0 kPmYvD"
              viewBox="0 0 24 24"
            >
              <path
                d="M9,22 L15,2 M17,17 L22,12 L17,7 M7,17 L2,12 L7,7"
                fill="none"
                stroke="#000"
                stroke-width="2"
              />
            </svg>
          </button>
          <div
            class="StyledBox__StyledBoxGap-sc-13pk1d4-1 hUZSMI"
          />
          <button
            class="StyledButton-sc-323bzc-0 kAlxhW"
            title="share"
            type="button"
          >
            <svg
              aria-label="Share"
              class="StyledIcon-ofa7kd-0 kPmYvD"
              viewBox="0 0 24 24"
            >
              <path
                d="M19,13 L19,23 L1,23 L1,5 L1,5 L11,5 M14,1 L23,1 L23,10 M10,14 L22.9999998,1 L10,14 Z"
                fill="none"
                stroke="#000"
                stroke-width="2"
              />
            </svg>
          </button>
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 kzkwJ"
        >
          <div
            class="StyledBox-sc-13pk1d4-0 isBQTF"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 klZxSv"
            >
              <label
                class="StyledBox-sc-13pk1d4-0 cacHrb"
                for="name"
              >
                <span
                  class="StyledText-sc-1sadyjn-0 hUokoe"
                >
                  name
                </span>
              </label>
              <div
                class="StyledBox-sc-13pk1d4-0 jJdKAu"
              >
                <div
                  class="StyledTextInput__StyledTextInputContainer-sc-1x30a0s-1 ksQrmr"
                >
                  <input
                    autocomplete="off"
                    class="StyledTextInput-sc-1x30a0s-0 klYlrf"
                    name="name"
                    style="text-align: right;"
                    value="my theme"
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            class="StyledBox-sc-13pk1d4-0 isBQTF"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 klZxSv"
            >
              <label
                class="StyledBox-sc-13pk1d4-0 cacHrb"
                for="rounding"
              >
                <span
                  class="StyledText-sc-1sadyjn-0 hUokoe"
                >
                  rounding
                </span>
              </label>
              <div
                class="StyledBox-sc-13pk1d4-0 jJdKAu"
              >
                <div
                  class="StyledBox-sc-13pk1d4-0 hErwNa"
                >
                  <input
                    class="StyledRangeInput-sc-15st9ck-0 fVQHiX"
                    id="rounding"
                    max="24"
                    min="0"
                    name="rounding"
                    step="2"
                    style="text-align: end;"
                    type="range"
                    value="4"
                  />
                  <div
                    class="StyledBox-sc-13pk1d4-0 jGYogm"
                  >
                    <div
                      class="StyledMaskedInput__StyledMaskedInputContainer-sc-99vkfa-1 eZPDpF"
                    >
                      <input
                        autocomplete="off"
                        class="StyledMaskedInput-sc-99vkfa-0 hIGWQk"
                        placeholder=""
                        style="text-align: right; width: 48px;"
                        value="4"
                      />
                    </div>
                    <span
                      class="StyledText-sc-1sadyjn-0 hUokoe"
                    >
                      px
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="StyledBox-sc-13pk1d4-0 isBQTF"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 klZxSv"
            >
              <label
                class="StyledBox-sc-13pk1d4-0 cacHrb"
                for="spacing"
              >
                <span
                  class="StyledText-sc-1sadyjn-0 hUokoe"
                >
                  spacing
                </span>
              </label>
              <div
                class="StyledBox-sc-13pk1d4-0 jJdKAu"
              >
                <div
                  class="StyledBox-sc-13pk1d4-0 hErwNa"
                >
                  <input
                    class="StyledRangeInput-sc-15st9ck-0 fVQHiX"
                    id="spacing"
                    max="36"
                    min="12"
                    name="spacing"
                    step="4"
                    style="text-align: end;"
                    type="range"
                    value="24"
                  />
                  <div
                    class="StyledBox-sc-13pk1d4-0 jGYogm"
                  >
                    <div
                      class="StyledMaskedInput__StyledMaskedInputContainer-sc-99vkfa-1 eZPDpF"
                    >
                      <input
                        autocomplete="off"
                        class="StyledMaskedInput-sc-99vkfa-0 hIGWQk"
                        placeholder=""
                        style="text-align: right; width: 48px;"
                        value="24"
                      />
                    </div>
                    <span
                      class="StyledText-sc-1sadyjn-0 hUokoe"
                    >
                      px
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="StyledBox-sc-13pk1d4-0 isBQTF"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 klZxSv"
            >
              <label
                class="StyledBox-sc-13pk1d4-0 cacHrb"
                for="scale"
              >
                <span
                  class="StyledText-sc-1sadyjn-0 hUokoe"
                >
                  font scale
                </span>
              </label>
              <div
                class="StyledBox-sc-13pk1d4-0 jJdKAu"
              >
                <div
                  class="StyledBox-sc-13pk1d4-0 hErwNa"
                >
                  <input
                    class="StyledRangeInput-sc-15st9ck-0 fVQHiX"
                    id="scale"
                    max="4"
                    min="0.2"
                    name="scale"
                    step="0.1"
                    style="text-align: end;"
                    type="range"
                    value="1"
                  />
                  <div
                    class="StyledBox-sc-13pk1d4-0 jGYogm"
                  >
                    <div
                      class="StyledMaskedInput__StyledMaskedInputContainer-sc-99vkfa-1 eZPDpF"
                    >
                      <input
                        autocomplete="off"
                        class="StyledMaskedInput-sc-99vkfa-0 hIGWQk"
                        placeholder=""
                        style="text-align: right; width: 48px;"
                        value="1"
                      />
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <button
            aria-label="Open Drop"
            class="StyledButton-sc-323bzc-0 eKdSxU"
            type="button"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 jLJwDx"
            >
              <span
                class="StyledText-sc-1sadyjn-0 hUokoe"
              >
                font
              </span>
              <div
                class="StyledBox__StyledBoxGap-sc-13pk1d4-1 hUZSMI"
              />
              <span
                class="StyledText-sc-1sadyjn-0 gmtNWw"
              >
                Helvetica
              </span>
            </div>
          </button>
          <button
            class="StyledButton-sc-323bzc-0 eKdSxU"
            type="button"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 jLJwDx"
            >
              <span
                class="StyledText-sc-1sadyjn-0 hUokoe"
              >
                colors
              </span>
            </div>
          </button>
          <button
            class="StyledButton-sc-323bzc-0 eKdSxU"
            type="button"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 jLJwDx"
            >
              <span
                class="StyledText-sc-1sadyjn-0 hUokoe"
              >
                form field
              </span>
            </div>
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`App empty 2`] = `
<div
  class="StyledGrommet-sc-19lkkz7-0 jKcNfZ"
>
  <div
    class="StyledGrid-sc-1wofa1l-0 bIzLvh"
  >
    <div
      class="StyledBox-sc-13pk1d4-0 vbgwq"
    >
      <div
        class="StyledBox-sc-13pk1d4-0 eYDYam"
      >
        <span
          class="StyledText-sc-1sadyjn-0 gmtNWw"
        >
          my theme
        </span>
        <button
          class="StyledButton-sc-323bzc-0 kAlxhW"
          title="edit theme"
          type="button"
        >
          <svg
            aria-label="Edit"
            class="StyledIcon-ofa7kd-0 kPmYvD"
            viewBox="0 0 24 24"
          >
            <path
              d="M14,4 L20,10 L14,4 Z M22.2942268,5.29422684 C22.6840146,5.68401459 22.6812861,6.3187139 22.2864907,6.71350932 L9,20 L2,22 L4,15 L17.2864907,1.71350932 C17.680551,1.319449 18.3127724,1.31277239 18.7057732,1.70577316 L22.2942268,5.29422684 Z M3,19 L5,21 M7,17 L15,9"
              fill="none"
              stroke="#000"
              stroke-width="2"
            />
          </svg>
        </button>
      </div>
      <div
        class="StyledBox-sc-13pk1d4-0 eXTSqT"
      >
        <div
          class="StyledBox-sc-13pk1d4-0 jJdKAu"
        >
          <label
            class="StyledRadioButton__StyledRadioButtonContainer-g1f6ld-0 eRuGHl"
            for="light"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 iOJnkv StyledRadioButton-g1f6ld-4 eLBYmH"
            >
              <input
                checked=""
                class="StyledRadioButton__StyledRadioButtonInput-g1f6ld-1 hBCEfd"
                id="light"
                name="themeMode"
                type="radio"
                value="light"
              />
              <div
                class="StyledBox-sc-13pk1d4-0 jxeStE StyledRadioButton__StyledRadioButtonBox-g1f6ld-3 fNAQJr"
              >
                <svg
                  class="StyledRadioButton__StyledRadioButtonIcon-g1f6ld-2 iHxKuy"
                  preserveAspectRatio="xMidYMid meet"
                  viewBox="0 0 24 24"
                >
                  <circle
                    cx="12"
                    cy="12"
                    r="6"
                  />
                </svg>
              </div>
            </div>
            <span>
              light
            </span>
          </label>
          <div
            class="StyledBox__StyledBoxGap-sc-13pk1d4-1 fskCS"
          />
          <label
            class="StyledRadioButton__StyledRadioButtonContainer-g1f6ld-0 eRuGHl"
            for="dark"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 iOJnkv StyledRadioButton-g1f6ld-4 eLBYmH"
            >
              <input
                class="StyledRadioButton__StyledRadioButtonInput-g1f6ld-1 hBCEfd"
                id="dark"
                name="themeMode"
                type="radio"
                value="dark"
              />
              <div
                class="StyledBox-sc-13pk1d4-0 kmXmMI StyledRadioButton__StyledRadioButtonBox-g1f6ld-3 fNAQJr"
              />
            </div>
            <span>
              dark
            </span>
          </label>
        </div>
      </div>
      <div
        class="StyledBox-sc-13pk1d4-0 llKfmK"
      >
        <div
          class="StyledBox-sc-13pk1d4-0 eXTSqT"
        >
          <h3
            class="StyledHeading-sc-1rdh4aw-0 jOHoby"
          >
            Guides
          </h3>
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 hFGUuI"
          tabindex="0"
        >
          Accessibility
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 hFGUuI"
          tabindex="0"
        >
          Typography
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 eXTSqT"
        >
          <h3
            class="StyledHeading-sc-1rdh4aw-0 jOHoby"
          >
            Examples
          </h3>
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 hFGUuI"
          tabindex="0"
        >
          Dashboard
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 hFGUuI"
          tabindex="0"
        >
          List
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 hFGUuI"
          tabindex="0"
        >
          Form
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 hFGUuI"
          tabindex="0"
        >
          Document
        </div>
      </div>
    </div>
    <div
      class="StyledBox-sc-13pk1d4-0 jJdKAu"
    />
    <div
      class="StyledBox-sc-13pk1d4-0 jiNNxm"
    >
      <div
        class="StyledBox-sc-13pk1d4-0 hTZIev"
      >
        <div
          class="StyledBox-sc-13pk1d4-0 koJzMg"
        >
          <button
            class="StyledButton-sc-323bzc-0 kAlxhW"
            title="choose another theme"
            type="button"
          >
            <svg
              aria-label="Apps"
              class="StyledIcon-ofa7kd-0 kPmYvD"
              viewBox="0 0 24 24"
            >
              <path
                d="M19,5 L21,5 L21,3 L19,3 L19,5 Z M11,5 L13,5 L13,3 L11,3 L11,5 Z M3,5 L5,5 L5,3 L3,3 L3,5 Z M19,13 L21,13 L21,11 L19,11 L19,13 Z M11,13 L13,13 L13,11 L11,11 L11,13 Z M3,13 L5,13 L5,11 L3,11 L3,13 Z M19,21 L21,21 L21,19 L19,19 L19,21 Z M11,21 L13,21 L13,19 L11,19 L11,21 Z M3,21 L5,21 L5,19 L3,19 L3,21 Z"
                fill="none"
                stroke="#000"
                stroke-width="2"
              />
            </svg>
          </button>
          <div
            class="StyledBox__StyledBoxGap-sc-13pk1d4-1 hUZSMI"
          />
          <div
            class="StyledBox-sc-13pk1d4-0 eitMOG"
          >
            <button
              class="StyledButton-sc-323bzc-0 jeaTYv"
              disabled=""
              title="undo last change"
              type="button"
            >
              <svg
                aria-label="Undo"
                class="StyledIcon-ofa7kd-0 kPmYvD"
                viewBox="0 0 24 24"
              >
                <path
                  d="M7.18,4,8.6,5.44,6.06,8h9.71a6,6,0,0,1,0,12h-2V18h2a4,4,0,0,0,0-8H6.06L8.6,12.51,7.18,13.92,2.23,9Z"
                />
              </svg>
            </button>
            <button
              class="StyledButton-sc-323bzc-0 jeaTYv"
              disabled=""
              title="redo last change"
              type="button"
            >
              <svg
                aria-label="Redo"
                class="StyledIcon-ofa7kd-0 kPmYvD"
                viewBox="0 0 24 24"
              >
                <path
                  d="M16.82,4,15.4,5.44,17.94,8H8.23a6,6,0,0,0,0,12h2V18h-2a4,4,0,0,1,0-8h9.71L15.4,12.51l1.41,1.41L21.77,9Z"
                />
              </svg>
            </button>
          </div>
          <div
            class="StyledBox__StyledBoxGap-sc-13pk1d4-1 hUZSMI"
          />
          <button
            class="StyledButton-sc-323bzc-0 kAlxhW"
            title="see JSON"
            type="button"
          >
            <svg
              aria-label="Code"
              class="StyledIcon-ofa7kd-0 kPmYvD"
              viewBox="0 0 24 24"
            >
              <path
                d="M9,22 L15,2 M17,17 L22,12 L17,7 M7,17 L2,12 L7,7"
                fill="none"
                stroke="#000"
                stroke-width="2"
              />
            </svg>
          </button>
          <div
            class="StyledBox__StyledBoxGap-sc-13pk1d4-1 hUZSMI"
          />
          <button
            class="StyledButton-sc-323bzc-0 kAlxhW"
            title="share"
            type="button"
          >
            <svg
              aria-label="Share"
              class="StyledIcon-ofa7kd-0 kPmYvD"
              viewBox="0 0 24 24"
            >
              <path
                d="M19,13 L19,23 L1,23 L1,5 L1,5 L11,5 M14,1 L23,1 L23,10 M10,14 L22.9999998,1 L10,14 Z"
                fill="none"
                stroke="#000"
                stroke-width="2"
              />
            </svg>
          </button>
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 kzkwJ"
        >
          <div
            class="StyledBox-sc-13pk1d4-0 isBQTF"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 klZxSv"
            >
              <label
                class="StyledBox-sc-13pk1d4-0 cacHrb"
                for="name"
              >
                <span
                  class="StyledText-sc-1sadyjn-0 hUokoe"
                >
                  name
                </span>
              </label>
              <div
                class="StyledBox-sc-13pk1d4-0 jJdKAu"
              >
                <div
                  class="StyledTextInput__StyledTextInputContainer-sc-1x30a0s-1 ksQrmr"
                >
                  <input
                    autocomplete="off"
                    class="StyledTextInput-sc-1x30a0s-0 klYlrf"
                    name="name"
                    style="text-align: right;"
                    value="my theme"
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            class="StyledBox-sc-13pk1d4-0 isBQTF"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 klZxSv"
            >
              <label
                class="StyledBox-sc-13pk1d4-0 cacHrb"
                for="rounding"
              >
                <span
                  class="StyledText-sc-1sadyjn-0 hUokoe"
                >
                  rounding
                </span>
              </label>
              <div
                class="StyledBox-sc-13pk1d4-0 jJdKAu"
              >
                <div
                  class="StyledBox-sc-13pk1d4-0 hErwNa"
                >
                  <input
                    class="StyledRangeInput-sc-15st9ck-0 fVQHiX"
                    id="rounding"
                    max="24"
                    min="0"
                    name="rounding"
                    step="2"
                    style="text-align: end;"
                    type="range"
                    value="4"
                  />
                  <div
                    class="StyledBox-sc-13pk1d4-0 jGYogm"
                  >
                    <div
                      class="StyledMaskedInput__StyledMaskedInputContainer-sc-99vkfa-1 eZPDpF"
                    >
                      <input
                        autocomplete="off"
                        class="StyledMaskedInput-sc-99vkfa-0 hIGWQk"
                        placeholder=""
                        style="text-align: right; width: 48px;"
                        value="4"
                      />
                    </div>
                    <span
                      class="StyledText-sc-1sadyjn-0 hUokoe"
                    >
                      px
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="StyledBox-sc-13pk1d4-0 isBQTF"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 klZxSv"
            >
              <label
                class="StyledBox-sc-13pk1d4-0 cacHrb"
                for="spacing"
              >
                <span
                  class="StyledText-sc-1sadyjn-0 hUokoe"
                >
                  spacing
                </span>
              </label>
              <div
                class="StyledBox-sc-13pk1d4-0 jJdKAu"
              >
                <div
                  class="StyledBox-sc-13pk1d4-0 hErwNa"
                >
                  <input
                    class="StyledRangeInput-sc-15st9ck-0 fVQHiX"
                    id="spacing"
                    max="36"
                    min="12"
                    name="spacing"
                    step="4"
                    style="text-align: end;"
                    type="range"
                    value="24"
                  />
                  <div
                    class="StyledBox-sc-13pk1d4-0 jGYogm"
                  >
                    <div
                      class="StyledMaskedInput__StyledMaskedInputContainer-sc-99vkfa-1 eZPDpF"
                    >
                      <input
                        autocomplete="off"
                        class="StyledMaskedInput-sc-99vkfa-0 hIGWQk"
                        placeholder=""
                        style="text-align: right; width: 48px;"
                        value="24"
                      />
                    </div>
                    <span
                      class="StyledText-sc-1sadyjn-0 hUokoe"
                    >
                      px
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="StyledBox-sc-13pk1d4-0 isBQTF"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 klZxSv"
            >
              <label
                class="StyledBox-sc-13pk1d4-0 cacHrb"
                for="scale"
              >
                <span
                  class="StyledText-sc-1sadyjn-0 hUokoe"
                >
                  font scale
                </span>
              </label>
              <div
                class="StyledBox-sc-13pk1d4-0 jJdKAu"
              >
                <div
                  class="StyledBox-sc-13pk1d4-0 hErwNa"
                >
                  <input
                    class="StyledRangeInput-sc-15st9ck-0 fVQHiX"
                    id="scale"
                    max="4"
                    min="0.2"
                    name="scale"
                    step="0.1"
                    style="text-align: end;"
                    type="range"
                    value="1"
                  />
                  <div
                    class="StyledBox-sc-13pk1d4-0 jGYogm"
                  >
                    <div
                      class="StyledMaskedInput__StyledMaskedInputContainer-sc-99vkfa-1 eZPDpF"
                    >
                      <input
                        autocomplete="off"
                        class="StyledMaskedInput-sc-99vkfa-0 hIGWQk"
                        placeholder=""
                        style="text-align: right; width: 48px;"
                        value="1"
                      />
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <button
            aria-label="Open Drop"
            class="StyledButton-sc-323bzc-0 eKdSxU"
            type="button"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 jLJwDx"
            >
              <span
                class="StyledText-sc-1sadyjn-0 hUokoe"
              >
                font
              </span>
              <div
                class="StyledBox__StyledBoxGap-sc-13pk1d4-1 hUZSMI"
              />
              <span
                class="StyledText-sc-1sadyjn-0 gmtNWw"
              >
                Helvetica
              </span>
            </div>
          </button>
          <button
            class="StyledButton-sc-323bzc-0 eKdSxU"
            type="button"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 jLJwDx"
            >
              <span
                class="StyledText-sc-1sadyjn-0 hUokoe"
              >
                colors
              </span>
            </div>
          </button>
          <button
            class="StyledButton-sc-323bzc-0 eKdSxU"
            type="button"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 jLJwDx"
            >
              <span
                class="StyledText-sc-1sadyjn-0 hUokoe"
              >
                form field
              </span>
            </div>
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
